devproxyrc.json

```diff
@@ -7,8 +7,7 @@
 			"pluginPath": "~appFolder/plugins/dev-proxy-plugins.dll",
 			"configSection": "objectsApi",
 			"urlsToWatch": [
-				"https://mkdemoapi.com/*",
-				"!https://login.microsoftonline.com/*"
```

Comment:
Since we don't want to intercept Entra calls, we can remove this Entry because they're not intercepted by default